buildbot/tox_setup.sh

```diff
@@ -1,74 +1,102 @@
 #!/usr/bin/env bash
-# pyenv root
-export PYENV_ROOT="$HOME/.pyenv"
-TEST_ROOT=$PWD/..
 
-# Install pyenv if it's missing
-if [[ ! -d $PYENV_ROOT ]]; then
-    git clone git://github.com/yyuu/pyenv.git ${PYENV_ROOT}
-    cd ${PYENV_ROOT}
-    # Get the latest tagged version
-    git checkout `git tag | tail -1`
+if [[ ! -d $PYENV_ROOT ]]
+then
+    export PYENV_ROOT="$HOME/.pyenv"
 fi
 
-# Upgrade it, if it's too old
-if [[ -z $(pyenv install --list | grep 3.4.3) ]]; then
-    cd ${PYENV_ROOT}
-    git pull origin master
-    git pull -u origin master
-    # Get the latest tagged version
-    git checkout `git tag | tail -1`
+declare -r PROJDIR="$PWD/.."
+if [[ ! -s $PROJDIR/riak/__init__.py ]]
+then
+    echo "[ERROR] script must be run from the root of a clone of github.com/basho/riak-python-client" 1>&2
```

Comment:
The only place `PROJDIR` is used is when running `pyenv local`. I don't want `pyenv global` to be used to set the python versions because that could screw someone's pyenv setup up (like it did mine). That means that `pyenv local` must be run in the correct place to create the `.python-version` file for the project.

An alternative would be to remove the `pyenv local` step and commit the `.python-version` file.